docs/intro/overview.rst

```diff
@@ -19,143 +19,221 @@ so you can get an idea of how it works and decide if Scrapy is what you need.
+they hidden inside ``a`` element:::
 
-   <p>
-   <strong>Category:</strong>
-   <a href="/cat/4">Movies</a> &gt; <a href="/sub/35">Documentary</a>
-   </p>
+    <a class="social-count" href="/scrapy/scrapy/network">
+        2,018
+    </a>
 
-   <p>
-   <strong>Total size:</strong>
-   150.62&nbsp;megabyte</p>
+.. highlight:: python
 
+Proper xpath to match this HTML would be:::
 
-.. highlight:: none
+    //a[@class='social-count']//text()
 
-An XPath expression to select the file size could be::
+At this point we have all we need to update our
+spider so that it will gather data from each repo. First we need to add
+item definitions to spider script, then we need to add new
+method to spider. Our new method will be called ``parse_repo`` and it is 
+going to be executed as callback for requests issued from ``parse``. 
+In this new method we will extract our data and return it.
 
-   //div[@id='specifications']/p[2]/text()[2]
+Here's the spider code::
 
-.. highlight:: python
+    from urlparse import urljoin
+
+    from scrapy import Spider, Item, Field
+    from scrapy.http import Request
+    from scrapy.contrib.loader import ItemLoader
 
-For more information about XPath see the `XPath reference`_.
 
-Finally, here's the spider code::
+    class RepoItem(Item):
+        url = Field()
+        name = Field()
+        forks = Field()
 
-    from scrapy.contrib.spiders import CrawlSpider, Rule
-    from scrapy.contrib.linkextractors import LinkExtractor
 
-    class MininovaSpider(CrawlSpider):
+    class GithubSpider(Spider):
+        name = 'github_spider'
+        start_urls = [
+            "https://github.com/trending?l=python"
+        ]
 
-        name = 'mininova'
-        allowed_domains = ['mininova.org']
-        start_urls = ['http://www.mininova.org/today']
-        rules = [Rule(LinkExtractor(allow=['/tor/\d+']), 'parse_torrent')]
+        def parse(self, response):
+            links_xpath = "//h3[@class='repo-list-name']/a/@href"
+            links = response.xpath(links_xpath).extract()[:4]
+            for link in links:
+                url = urljoin(response.url, link)
+                yield Request(url, callback=self.parse_repo)
 
-        def parse_torrent(self, response):
-            torrent = TorrentItem()
-            torrent['url'] = response.url
-            torrent['name'] = response.xpath("//h1/text()").extract()
-            torrent['description'] = response.xpath("//div[@id='description']").extract()
-            torrent['size'] = response.xpath("//div[@id='info-left']/p[2]/text()[2]").extract()
-            return torrent
+        def parse_repo(self, response):
+            loader = ItemLoader(item=RepoItem(), selector=response)
+            loader.add_value("url", response.url)
+            loader.add_xpath("name", "//a[@class='js-current-repository']/text()")
+            loader.add_xpath("forks", "//a[@class='social-count']//text()")
+            yield loader.load_item()
```

Comment:
+1, better to avoid item loaders on first exposure -- for the same reasons as I previously mentioned (less things for user to know/learn, quicker she'll get things done and feel badass).
If #1064 gets implemented, we might as well yield dicts here too.

User can learn about item loaders, XPath or whatever, later.